tests/regression/77-lin2vareq/13-join-non-constant.c

```diff
@@ -0,0 +1,22 @@
+// SKIP PARAM: --set ana.activated[+] lin2vareq --set ana.relation.privatization top --set sem.int.signed_overflow assume_none
+
+int main(void) {
+  int a, b, c, d;
+
+  int t;
+  if (t) {
+    b = a + 2;
+    c = a + 7;
+    d = a + 1;
+  } else {
+    b = a + 30;
+    c = a + 35;
+    d = a + 10;
+  }
+  __goblint_check(c == b + 5); // SUCCESS
+  __goblint_check(c == b + 3); // FAILURE
+  __goblint_check(d == b + 3); // UNKNOWN!
```

Comment:
Can this assertion actually be true? An `UNKNOWN!` annotation means that an assertion may both be `true` and `false` for some concrete executions.